src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null && repeatPassword == null) {
```

Comment:
we don't need 3 separate if clause, keep your code simple, let's solve the task with one if-condition